src/lib/schematics/update/material/data/property-names.ts

```diff
@@ -335,5 +335,18 @@ export const propertyNames = transformChanges<MaterialPropertyNameData>([
         }
       }
     ]
+  },
+
+  {
```

Comment:
Please add a comment `// TODO(devversion): this should be part of the V6 to V7 upgrade`.